src/core/useTrackBindingPlugin.js

```diff
@@ -82,8 +78,6 @@ export class TrackBindingPlugin {
         if (eventName) {
             delete dataset.eventName;
             callback(eventName, dataset, mergePagedefaults === "true");
-        } else {
-            callback(dataset, mergePagedefaults === "true");
         }
```

Comment:
Since some people may rely on this to still trigger without an `eventName`, should you throw an `invariant` or `warning` here? If so, could you also add a test that verifies it throws an `invariant`?

I believe this change will require a version bump to 2.3.0 since some may expect it to still trigger? 

I looked through the docs to see if it's mentioned that `data-metrics-event-name` is required, I haven't found an issue yet but wanted to verify